libs/base/eventcontext.ts

```diff
@@ -77,7 +83,12 @@ namespace control {
             control.enablePerfCounter("all frame callbacks")
 
             let loopStart = control.millis()
-            this.deltaTimeMillis = loopStart - this.prevTimeMillis;
+            if (_animationTime) {
+                this.deltaTimeMillis = 30;
```

Comment:
What’s the meaning of 30?

Reply:
This is what we typically get without this (ie 31 fps or so)